jenkins/e2e-image/e2e-runner.sh

```diff
@@ -27,6 +27,7 @@ mkdir -p ${ARTIFACTS}
 : ${KUBE_GCS_RELEASE_BUCKET:="kubernetes-release"}
 : ${KUBE_GCS_DEV_RELEASE_BUCKET:="kubernetes-release-dev"}
 : ${JENKINS_SOAK_PREFIX:="gs://kubernetes-jenkins/soak/${JOB_NAME}"}
+: ${JENKINS_FEDERATION_PREFIX:="gs://kubernetes-jenkins/federation/${JOB_NAME}"}
```

Comment:
I would like to keep it separate. That way it is clear where all the federation kubeconfig files are.